docs/securecore.butane

```diff
@@ -0,0 +1,42 @@
+variant: fcos
```

Comment:
can we rename this to example.butane

so people don't mistakenly think that this is required?

Reply:
yes